app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @like = Like.new(post_id: @post.id, user_id: @user.id)
+
+    respond_to do |format|
+      format.html { redirect_to request.referer } if @like.save
+    end
+  end
```

Comment:
- [ ] **[OPTIONAL]** The previous reviewer could not create a like. But in my case, after the first like, I could not create another like for the same post again. I would recommend you fix this or handle the error in a more efficient manner, instead of having the app crash every time.

  ![image](https://github.com/Abdulwhbhussain/blog-app/assets/36542986/2699c952-a02f-405b-a51a-46dbf3f6dee1)

Reply:
Sure Brother!